templates/csharp/message-extension-with-api-from-scratch-api-key/appPackage/responseTemplates/repair.json

```diff
@@ -1,6 +1,6 @@
 {
   "version": "1.0",
-  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.19/MicrosoftTeams.ResponseRenderingTemplate.schema.json",
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.21/MicrosoftTeams.ResponseRenderingTemplate.schema.json",
```

Comment:
should we change this version?

Reply:
not sure. what bad effect will be if we upgrade this?

Reply:
Please double confirm with PM on what should be changed, seems we only talked about upgrading the app manifest version?
Let's avoid making additional changes in the service release.

Reply:
Agree. I've talked with template owner @huimiu . He confirmed that if no new feature is needed, this version doesn't need to be upgrade as the manifest version.
I pushed back the changes.